Cargo.toml

```diff
@@ -0,0 +1,72 @@
+[package]
+name = "libtypec-rs"
+authors = ["Rajaram Regupathy"]
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+argh = "0.1.12"
```

Comment:
The CLI should be in a separate crate so that downstream consumers don't get stuck depending on `argh`.

Reply:
@refi64 what about cargo workspaces?